app/models/application_record.rb

```diff
@@ -2,6 +2,6 @@ class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
   def self.random
-    offset(rand(0..self.count)).first
+    order("RANDOM()").first
```

Comment:
this should return a record every time now :) Now we don't have to worry about it returning nil for the first or last random number.